src/components/UserProfile/BlueSquares/BlueSquare.jsx

```diff
@@ -6,7 +6,8 @@ const BlueSquare = ({ blueSquares, handleBlueSquare, role, roles, userPermission
   return (
     <div className="blueSquareContainer">
       <div className="blueSquares">
-        {blueSquares ? (blueSquares.map((blueSquare, index) => (
+        {blueSquares ? (blueSquares.sort((a,b)=> a.date > b.date ? 1 : -1)
```

Comment:
Looks Good! Tested it out locally and it works. The most recent blue recent blue squares show up in the right and the least recent at the left.